.github/workflows/docker.yml

```diff
@@ -12,6 +12,7 @@ on:
 
 permissions:
   contents: read
+  packages: write
```

Comment:
Not needed since we only push with the `BrewTestBot` token.

Reply:
@reitermarkus I will remove but can you please try to avoid having different feedback in each round of review and provide a "✅ with comments" unless you are confident this is going to blow up in `master`?

Reply:
Sure, will do.

Previously there was `cache-to` using the `github-actions[bot]` token and the `packages: write` permission was correct. Now it's only using `cache-from`, so of course my feedback is different depending on the current state of changes.

Reply:
@reitermarkus Sure, makes sense. Given the amount of time between rounds of review it'd be helpful to have a "if you do X, please remove Y". Regardless: thanks for the review ❤️